src/foam/doc/DigSelectView.js

```diff
@@ -0,0 +1,134 @@
+foam.CLASS({
+  package: 'foam.doc',
+  name: 'DigSelectView',
+  extends: 'foam.u2.View',
+  implements: [
+    'foam.mlang.Expressions',
+  ],
+  imports: [
+    'appConfig',
+    'user',
+  ],
+  requires: [
+    'foam.doc.CodeTabs',
+    'foam.u2.Tab',
+    'foam.doc.dao.AxiomDAO',
```

Comment:
Remove any unused requires. AxiomDAO, PropertyAxiom, CodeTab, DetailView, PromiseSlot

Reply:
I'm not exactly sure how I know if something is used or not .. there is no clear referencing mechanism, beside checking where fields come through text search. I'll just play around with removing things and if they break I'll add them back I guess.

Reply:
was pretty simple after all .. just removed everything that wasn't referenced in a 'self.' reference.